x86/bell/server.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+var express = require('express')
+var PortJack = express()
+var path = require("path");
+
+PortJack.get(/^(.+)$/, function(req, res) {
+    var options = {
+        "127.0.0.1": "http://127.0.0.1:2200/apps/_design/bell/MyApp/index.html",
+        "localhost": "http://localhost:2200/apps/_design/bell/MyApp/index.html"
+    }
+
+    if (options.hasOwnProperty(req.hostname)) {
+        res.setHeader('Location', options[req.hostname])
+    } else {
+        res.setHeader('Location', 'http://docker.ole.org:2200/apps/_design/bell/MyApp/index.html')
```

Comment:
The address should set by dynamically and specified by the environment variable.